kde-apps/knetwalk/knetwalk24-24.05.0.recipe

```diff
@@ -0,0 +1,135 @@
+	lib:libKF6ConfigWidgets$secondaryArchSuffix
+	lib:libKF6CoreAddons$secondaryArchSuffix
+	lib:libKF6Crash$secondaryArchSuffix
+	lib:libKF6DBusAddons$secondaryArchSuffix
+	lib:libKF6GuiAddons$secondaryArchSuffix
+	lib:libKF6I18n$secondaryArchSuffix
+	lib:libKF6IconThemes$secondaryArchSuffix
+	lib:libKF6ItemViews$secondaryArchSuffix
+	lib:libKF6JobWidgets$secondaryArchSuffix
+	lib:libKF6KIOCore$secondaryArchSuffix
+	lib:libKF6Notifications$secondaryArchSuffix
+	lib:libKF6NotifyConfig$secondaryArchSuffix
+	lib:libKF6Package$secondaryArchSuffix
+	lib:libKF6SonnetUi$secondaryArchSuffix
+	lib:libKF6TextWidgets$secondaryArchSuffix
+	lib:libKF6WidgetsAddons$secondaryArchSuffix
+	lib:libKF6WindowSystem$secondaryArchSuffix
+	lib:libKF6XmlGui$secondaryArchSuffix
+	lib:libQt6Core$secondaryArchSuffix
+	lib:libQt6DBus$secondaryArchSuffix
+	lib:libQt6Gui$secondaryArchSuffix
+	lib:libQt6Qml$secondaryArchSuffix
+	lib:libQt6Widgets$secondaryArchSuffix
+	lib:libQt6Xml$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	extra_cmake_modules$secondaryArchSuffix
+	devel:libKDEGames6$secondaryArchSuffix
+	devel:libKF6AuthCore$secondaryArchSuffix
+	devel:libKF6Codecs$secondaryArchSuffix
+	devel:libKF6ColorScheme$secondaryArchSuffix
+	devel:libKF6Completion$secondaryArchSuffix
+	devel:libKF6ConfigCore$secondaryArchSuffix
+	devel:libKF6ConfigWidgets$secondaryArchSuffix
+	devel:libKF6CoreAddons$secondaryArchSuffix
+	devel:libKF6Crash$secondaryArchSuffix
+	devel:libKF6DBusAddons$secondaryArchSuffix
+	devel:libKF6DocTools$secondaryArchSuffix
+	devel:libKF6GuiAddons$secondaryArchSuffix
+	devel:libKF6I18n$secondaryArchSuffix
+	devel:libKF6IconThemes$secondaryArchSuffix
+	devel:libKF6ItemModels$secondaryArchSuffix
+	devel:libKF6ItemViews$secondaryArchSuffix
+	devel:libKF6JobWidgets$secondaryArchSuffix
+	devel:libKF6KIOCore$secondaryArchSuffix
+	devel:libKF6Notifications$secondaryArchSuffix
+	devel:libKF6NotifyConfig$secondaryArchSuffix
+	devel:libKF6Package$secondaryArchSuffix
+	devel:libKF6TextWidgets$secondaryArchSuffix
+	devel:libKF6WidgetsAddons$secondaryArchSuffix
+	devel:libKF6WindowSystem$secondaryArchSuffix
+	devel:libKF6XmlGui$secondaryArchSuffix
+	devel:libQt6Core$secondaryArchSuffix
+	devel:libQt6Qml$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+PATCH()
+{
+	# disable docs generation
+	sed -e '/add_subdirectory(doc)/ s/^#*/#/' -i CMakeLists.txt
+	sed -e '/kdoctools_install(po)/ s/^#*/#/' -i CMakeLists.txt
+	# remove dbus service
+	sed -e '/KDBusService service;/d' -i src/main.cpp
+}
+
+BUILD()
+{
+	cmake -Bbuild -S. $cmakeDirArgs \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DECM_DIR=/system/data/cmake/Modules/ECM/cmake
+	make -Cbuild $jobArgs
+}
+
+INSTALL()
+{
+	make -Cbuild install
+
+	# cleanup
+	rm -rf $dataDir/{applications,metainfo}
+
+	mkdir -p "$dataDir/deskbar/menu/Applications/KDE"
+
+	local APP_SIGNATURE="application/x-vnd.kde-kdiamond"
```

Comment:
This is the wrong game ;-)

Reply:
Missed that, thanks! Should revbump it also for the Deskbar menu entry. ;)

Reply:
Fixed in: https://github.com/haikuports/haikuports/pull/10593